ADOPTERS.md

```diff
@@ -0,0 +1,35 @@
+# Confidential Containers Adopters
+
+This page contains a list of organizations/companies and projects/products who use confidential containers as adopters in different usage levels (development, beta, production, GA etc...) 
+
+**NOTE:** For adding your organization/company and project/product to this table (alphabetical order) fork the repository and open a PR with the required change. 
+See list of adopter types at the bottom of this page.
```

Comment:
```suggestion
See list of adopter types in [Adopter Types][1].

[1]: #adopter-types
```

Then we can have a hyperlink of markdown.

Reply:
Any chance we can differ the change to after the basic page was merged?
I'm not familiar with hyperlink markdown within the page and I won't really have time today to look into it and make sure it works